pkgs/development/libraries/science/chemistry/openmm/default.nix

```diff
@@ -6,10 +6,16 @@
 , fftwSinglePrec
 , doxygen
 , swig
-, python3Packages, enablePython ? false
-, opencl-headers, ocl-icd, enableOpencl ? false
-, clang, enableClang ? true
-, cudatoolkit, enableCuda ? false
+, python3Packages
```

Comment:
Nitpick: can we re-order these: switch first, corresponding dependencies below?

Reply:
Sure :)